src/utils/TxmaEvent.ts

```diff
@@ -1,18 +1,21 @@
-import { VerifiedCredential } from "./IVeriCredential";
+import { DocumentDetails, DrivingPermit, VerifiedCredential, VerifiedCredentialEvidence, VerifiedCredentialSubject } from "./IVeriCredential";
 import { ISessionItem } from "../models/ISessionItem";
 import { absoluteTimeNow } from "./DateTimeUtils";
+import { PostOfficeInfo } from "../models/YotiPayloads";
 
 export type TxmaEventName =
 	"F2F_CRI_START"
 	| "F2F_CRI_AUTH_CODE_ISSUED"
 	| "F2F_YOTI_PDF_EMAILED"
 	| "F2F_YOTI_START"
-	| "F2F_YOTI_END"
-	| "F2F_CRI_VC_ISSUED";
+	| "F2F_YOTI_RESPONSE_RECEIVED"
+	| "F2F_CRI_VC_ISSUED"
+	| "F2F_CRI_END";
+
 
 export interface TxmaUser {
 	"user_id": string;
-	"transaction_id": string;
+	"transaction_id": string | undefined;
```

Comment:
Out of curiosity why have we done `string | undefined` rather than `"transaction_id"? : string`

Reply:
When I first started working on it, `"transaction_id"? : string` threw errors but it works now, so it's all changed